src/API/Recruit.Api.UnitTests/Controllers/TraineeshipVacanciesControllerTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Net;
+using System.Threading.Tasks;
+using FluentAssertions;
+using SFA.DAS.Recruit.Api.Controllers;
+using Xunit;
+using Moq;
+using MediatR;
+using SFA.DAS.Recruit.Api.Queries;
+using System.Threading;
+using AutoFixture.NUnit3;
+using Esfa.Recruit.Vacancies.Client.Domain.Entities;
+using Microsoft.AspNetCore.Mvc;
+using NUnit.Framework;
+using SFA.DAS.Recruit.Api.Commands;
+using SFA.DAS.Recruit.Api.Models;
+using SFA.DAS.Testing.AutoFixture;
+using Assert = Xunit.Assert;
+using Esfa.Recruit.Vacancies.Client.Application.Configuration;
+
+namespace SFA.DAS.Recruit.Api.UnitTests.Controllers
+{
+    public class TraineeshipVacanciesControllerTests
```

Comment:
This probably wants to exist in the VacanciesControllerTests - the action is part of that controller - this is implying there is a different controller